applications/xr_gsplat/README.md

```diff
@@ -0,0 +1,70 @@
+# XR + Gaussian Splatting
+
+This application demonstrates rendering a 3D scene using Gaussian Splatting in XR.  
+
+### 0. Training a Gaussian Splatting Model
```

Comment:
Thanks, these instructions look clear. Would be nice to move these to an automated preprocessing step, not a blocker for this PR though